### PR TITLE
Use more accessible tooltip for errata correction schedule

### PR DESCRIPTION
**Pull request by RoyEJohnson**
Need to pull data from CMS

### CODE REVIEW

File: src/app/pages/errata-summary/hero/hero.js [hunk 1]
```diff
@@ -34,29 +29,80 @@ function useBookInfo(book) {
     return info;
 }
 
-function useErrataHoverHtml(slug) {
-    const [html, setHtml] = useState('<p>...loading...</p>');
+const middle = '-135';
+
+function shiftIntoView(ref, leftOffset, setLeftOffset) {
+    const {left, right} = ref.current.getBoundingClientRect();
+    const {right: pageRight} = ref.current.closest('.page').getBoundingClientRect();
```
Comment (nathanstitt) [inline src/app/pages/errata-summary/hero/hero.js:36]: what's your browser support like?  will it have `closest` ?  ie 11 doesn't have that

File: src/app/pages/errata-summary/hero/hero.js [hunk 2]
```diff
@@ -34,29 +29,80 @@ function useBookInfo(book) {
...
+        <div className="poptip-attachment">
+            <div className="poptip" ref={ref} style={style} role="tooltip" aria-hidden={!isOpen}>
+                <div className="title-bar">
+                    More information
+                </div>
+                <RawHTML className="body" html={html} />
+            </div>
+        </div>
+    );
+}
```
Comment (nathanstitt) [inline src/app/pages/errata-summary/hero/hero.js:69]: is this the wrapper element?  if so should it have `aria-alert` set?

**Review by nathanstitt — APPROVED** [review]

File: src/app/pages/errata-summary/hero/hero.js (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/pages/errata-summary/hero/hero.js:36]: We don't support IE, only Edge. Good point, though.

File: src/app/pages/errata-summary/hero/hero.js (hunk 2 above)
Comment (RoyEJohnson) [reply src/app/pages/errata-summary/hero/hero.js:69]: This is not an alert. I have set role to `tooltip`.

Comment (nathanstitt) [reply src/app/pages/errata-summary/hero/hero.js:69]: seems like role="tooltip" does the same though, so never mind!

**Pull request merged**